test/connectors/mocks/MockCreateTrnService.scala

```diff
@@ -17,7 +17,7 @@
 package connectors.mocks
```

Comment:
Should this class be in the package services.mocks rather than connectors.mocks?